### DISCUSSION TITLE
Will this work on Mint? (or any other Ubuntu/Debian based OS)

### DISCUSSION THREAD

**Discussion by piotrek-k** [regolith-linux/regolith-desktop#945] [Q&A] [answered]
Installation steps assume that user uses either Ubuntu or Debian, but is it because it cannot be installed elsewhere or because other distros are not officially supported? Does it require some specific dependencies that are available on Ubuntu/Debian only?

**Comment by DanRunfola** [comment]
I ran 2.* on top of mint for ages, no problems at all.

Thinking about giving version 3 a go over the winter holiday.

**Comment by kgilmer** [comment, marked as answer]
Sorry, missed this earlier.  it is because, mainly, of lack of users that can test.  But also, I could be wrong but I would expect the version of major components such as GTK and GNOME would differ from Ubuntu such that managing the dependencies, while not particularly difficult, adds considerable overhead to work required to release.

There is no policy or anything that would prevent Regolith running on any Debian-based system that was able to resolve the transitive set of Regolith Debian package dependencies as specified in the Regolith `apt` repository URL.  

In summary, give it a shot and let us know how it works out.  :)
